lib/metasearch_index/inserter.rb

```diff
@@ -24,10 +24,18 @@ def document
         {
           exact_query: @document["exact_query"],
           stemmed_query: @document["stemmed_query"],
-          stemmed_query_as_term: @document["stemmed_query_as_term"],
+          stemmed_query_as_term: @document["stemmed_query"].presence && " #{analyzed_stemmed_query} ",
```

Comment:
is this the same as `" #{analyzed_stemmed_query} if @document["stemmed_query"]`?

Also, how come there's spaces around it?